operator/website/config.toml

```
@@ -4,4 +4,13 @@ title = 'Loki Operator'
 theme = 'doks'
 
 [permalinks]
-  docs = "/:sections/:filename.md"
\ No newline at end of file
+  docs = "/:sections/:filename.md"
+
+[markup]
+  [markup.goldmark]
+    [markup.goldmark.renderer]
+      unsafe = true
+  [markup.tableOfContents]
+    endLevel = 4
+    ordered = false
+    startLevel = 1
```

As far as I know, the indentations and "nested section headers" are purely decorational in toml, so this could be simplified to this, if we want to:

```suggestion
[markup.goldmark.renderer]
  unsafe = true
[markup.tableOfContents]
  endLevel = 4
  ordered = false
  startLevel = 1
```

We could even leave out the indentation of the sections completely.